### PR TITLE
T3T1: tutorial flow

### LINKED ISSUE #3829: Tutorial flow
**Design:** [Figma](https://www.figma.com/design/n43caWJFGJlT167AQcxruu/%E3%80%94-Prod-%E3%80%95Safe-5?m=auto&node-id=5220%3A1963&t=u1Te0KuammAazwOm-1)

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #3829.

Missing, to be done by end of week:
- [x] color and icon on the first screen
- [x] color of the hold-to-confirm screen on the "cancel" path
- [ ] will it fit in flash?
- [x] update translated copy

emu00000000 emu00000001 emu00000003 emu00000004 emu00000005 emu00000006 emu00000007

### CODE REVIEW

**Review by obrusvit — COMMENTED** [review]
Code is pretty straightforward.
We need to handle changed translations.

**Comment by Hannsek** [comment]
Yes, there will be also some copy minor tweaks.

**Comment by Hannsek** [comment]
@mmilata sorry, we've updated the copy a bit. Please go through it [here](https://www.figma.com/design/n43caWJFGJlT167AQcxruu/%E3%80%94-Prod-%E3%80%95Safe-5?m=auto&node-id=5220-1963&t=GzQ7Uem7cRBDENDG-1) and change it accordingly.

**Comment by mmilata** [comment]
Fixed the colors and copy in [1fe3569](https://github.com/trezor/trezor-firmware/pull/3982/commits/1fe3569a0471e9d9491ee145699acf54bf52ed97), please take a look.

File: core/translations/en.json
Comment (obrusvit) [inline core/translations/en.json:1]: The following strings can be removed (please remove also from `de/es/fr/cs.json`)
- `instructions__hold_to_finish_tutorial`
- `tutorial__first_transaction_intro`
- `tutorial__first_transaction_outro`
- `tutorial__one_more_step`
- `tutorial__subtitle_safe5`

**Review by obrusvit — CHANGES_REQUESTED** [review]

**Review by obrusvit — APPROVED** [review]
Tested ACK.

Limitations not blocking this PR:
- HoldToConfirm danger, the filling circle is green, should probably be the same color as the danger circle
- removing the obsolete strings as my previous comment
- other langs (in progress)

**Comment by Hannsek** [comment]
> HoldToConfirm danger, the filling circle is green, should probably be the same color as the danger circle

This is the same on other places as well, like Hold to wipe. Let's fix it on all places at once.

File: core/translations/en.json
Comment (obrusvit) [reply core/translations/en.json:1]: https://github.com/trezor/trezor-firmware/pull/3982/commits/0185544aec44ee975d0b0fa2ff96808e91927e60

**Comment by bosomt** [comment]
**QA OK**

`trezorctl device tutorial`


noticed red color when canceling

image

and differently pointing welcome screen

image


 - Device: Trezor T3T1 2.8.0 regular (revision dd4671a5104952ef505d28d1f9e94d1484b4607a)

**Comment by Hannsek** [comment]
Both are expected.

**Pull request merged**